angular/core/models/event_model.coffee

```diff
@@ -4,53 +4,24 @@ angular.module('loomioApp').factory 'EventModel', (BaseModel) ->
     @plural: 'events'
     @indices: ['id', 'discussionId']
 
-    @eventTypeMap: {
-      'user_added_to_group':            'group',
-      'membership_request_approved':    'group',
-      'discussion_moved' :              'group',
-      'membership_requested':           'membershipRequest',
-      'new_discussion':                 'discussion',
-      'discussion_edited':              'discussion',
-      'new_motion':                     'proposal',
-      'motion_closed':                  'proposal',
-      'motion_closed_by_user':          'proposal',
-      'motion_edited':                  'proposal',
-      'new_vote':                       'proposal',
-      'motion_closing_soon':            'proposal',
-      'motion_outcome_created':         'proposal',
-      'new_comment':                    'comment',
-      'comment_liked':                  'comment',
-      'comment_replied_to':             'comment',
-      'user_mentioned':                 'comment',
-      'invitation_accepted':            'membership',
-      'new_coordinator':                'membership'
-    }
+    @eventTypeMap:
+      group:              'groups'
```

Comment:
Simplifies the event lookup a lot by going off of what kind of eventable is is, rather than what event kind it is.

(this says, 'if the event type is discussion, use Records.discussions to find it by eventable id)